staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// KubeSchedulerConfiguration xx
+type KubeSchedulerConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// address is the IP address for the Kubelet to serve on
+	Address string `json:"address,omitempty"`
```

Comment:
Should align comment‘ format , using `Address` ? The same other places

Reply:
other way around... comments should match the serialized values so the generated doc matches the files people will write